### ISSUE TITLE
Question: which tool produced all these models?

### ISSUE THREAD

**Issue by 5nizza** [meyerphi/syntcomp-reference#1]
hi, were all these models produced by strix or by other tools as well? (it might be worth adding a name of the tool that produced the aag model, e.g. as a comment in the aag file). Useful repo, thanks!

**Comment by meyerphi** [comment]
The models are from different sources. I added a small remark to the README, let me expand on that here.

Some models are taken as the minimal ones created by past SYNTCOMP participants, among them Strix, ltlsynt, BoSy, BoWser, Party and Acacia. These were however also postprocessed with ABC and additional minimization commands. Many models are also handwritten from scratch, as there are often much smaller solutions than any of the tools can produce. Unfortunately, now it is impossible for me to discern which solution was obtained in which way, so I cannot annotate them.

My goal with this repository was not to compare different tools, which is already done by SYNTCOMP, and can be done by just running the tools on your own. I rather have the two following goals:
- Provide for each specification a model that can be verified within a reasonable time, so there is a complete classification into realizable and unrealizable specifications.
- Provide models as small as possible (minimal would be ideal, but hard to obtain), by a reasonable "best effort" practice. For this, I included handwritten models, even if no tool could produce such a small model. These models can then be seen by tool developers as a reference to strive for, and could maybe be obtainable by improved algorithms eventually.

This also led to some insights about past results: one specification was incorrectly classified as realizable (by Acacia), but had no model to prove it, and some models produced by BoWSer were actually not minimal, even though the tool claimed to produce minimal solutions (unfortunately impossible to reproduce as BoWSer was never published).
